clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -269,4 +274,29 @@ public static void waitForCondition(final TestCondition testCondition, final lon
         }
     }
 
-}
+    /**
+     * Checks if a cluster id is valid.
+     * @param clusterId
+     * @return true if valid, false otherwise.
+     */
+    public static boolean isValidClusterId(String clusterId) {
```

Comment:
Maybe this method can use JUnit assertions for better error messages (instead of returning `boolean`).